great_expectations/data_context/data_context.py

```diff
@@ -933,7 +931,8 @@ def delete_expectation_suite(self, expectation_suite_name):
         """
         expectation_suite = ExpectationSuite(expectation_suite_name=expectation_suite_name)
         key = ExpectationSuiteIdentifier(expectation_suite_name=expectation_suite_name)
-
+        del self._stores[self.expectations_store_name][key]
+        return True
         if not self._stores[self.expectations_store_name].has_key(key):
             raise ge_exceptions.DataContextError(
                 "expectation_suite with name {} does not exist."
```

Comment:
Code after the return won't be reachable.

Reply:
Refresh; i tend to keep pushing so as to not lose work.

Reply:
WIP at the time. Looks resolved.